css-syntax/styles.css

```diff
@@ -0,0 +1,33 @@
+body {
+  color: #090902;
+  font-family: sans-serif;
+}
+
+h1 {
+  text-align: center;
+}
+
+* article {
+  padding-left: 16px;
+  padding-right: 16px;
```

Comment:
You don't need to have the `*` here. If you have them together it is targeting all `article` elements that are a child of any element. While this is true since it would always be a child of at least a `body` element, it is inefficient to use and not as good for code readability.